drf_extra_fields/fields.py

```diff
@@ -180,23 +180,31 @@ def to_internal_value(self, data):
         """
         if html.is_html_input(data):
             data = html.parse_html_dict(data)
+
         if not isinstance(data, dict):
             self.fail('not_a_dict', input_type=type(data).__name__)
+
+        extra_content = list(set(data) - set(["lower", "upper", "bounds", "empty"]))
```

Comment:
Instead of calling `set()` on a list, we can directly say `{"lower", "upper", "bounds", "empty"}`

Reply:
This is a set difference operation and the first argument have to include `set()` since `{dict()}` is not valid. I have used `set(list())` for the second argument to be consistent with the first argument. By the way, `set(list())` is also a common practice.

Reply:
Here, I initially proposed `extra_content` to be `list(set(data) - {"lower", "upper", "bounds", "empty"})`, however, on second thought, I agree that doing `set() - set()` makes it more readable.